repos/clone.sh

```diff
@@ -3,7 +3,19 @@ set -e
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 
-for repo in hoover/snoop2 hoover/search hoover/ui liquidinvestigations/core liquidinvestigations/authproxy hypothesis/h; do (
+if [ "$1" == 'ssh' ]; then
+        echo "using ssh"
+elif [ "$1" == 'https' ]; then
+        echo "using https"
+else
+        set +x
+        echo "Usage: $0 https - clone repos with https"
+        echo "       $0 ssh   - clone repos with ssh"
+        exit 1
+fi
+
+for repo in liquidinvestigations/hoover-snoop2 liquidinvestigations/hoover-search liquidinvestigations/hoover-ui liquidinvestigations/core liquidinvestigations/authproxy liquidinvestigations/hypothesis-h; do
```

Comment:
Since you're here, can you please make this list into a real list?

```bash
repos=(
  hoover-snoop2
  hoover-search
  ...
)

for repo in "${repos[@]}"; do
...
done
```

Reply:
ok